ports/vcpkg-tool-swig/portfile.cmake

```
@@ -0,0 +1,30 @@
+set(VCPKG_BUILD_TYPE "release")
+set(VCPKG_POLICY_EMPTY_INCLUDE_FOLDER enabled)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO swig/swig 
+    REF "v${VERSION}"
+    SHA512 5d653333f73356d4d5ba8b615882e49f33f188bc68d8204352116bc4aca7946ec01ce2e02524c5ce805b98c2219ed05e664120485bf18095c5c0785436487074
+    HEAD_REF master
+)
+
+vcpkg_find_acquire_program(BISON)
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        "-DBISON_EXECUTABLE=${BISON}"
+)
+
+vcpkg_cmake_install()
+vcpkg_fixup_pkgconfig()
+
+vcpkg_copy_tools(
+    TOOL_NAMES swig
+    DESTINATION "${CURRENT_PACKAGES_DIR}/tools/swig"
+)
+
+file(COPY "${CURRENT_PACKAGES_DIR}/bin/" DESTINATION "${CURRENT_PACKAGES_DIR}/tools/swig")

```

```suggestion
vcpkg_copy_tools(TOOL_NAMES swig AUTO_CLEAN)

```
Note the `AUTO_CLEAN`.